workers/unity/Assets/Playground/Scripts/Worker/AndroidClientWorkerConnector.cs

```diff
@@ -0,0 +1,60 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Mobile;
+#if UNITY_ANDROID
+using Improbable.Gdk.Mobile.Android;
+#endif
+using System;
+using UnityEngine;
+
+
+namespace Playground
+{
+    public class AndroidClientWorkerConnector : MobileWorkerConnector, IMobileClient
+    {
+        public string IpAddress { get; set; }
+        public ConnectionScreenController ConnectionScreenController { get; set; }
+
+        [SerializeField] private GameObject level;
+
+        private GameObject levelInstance;
+
+        public async void TryConnect()
+        {
+            await Connect(WorkerUtils.AndroidClient, new ForwardingDispatcher()).ConfigureAwait(false);
+        }
+
+        protected override void HandleWorkerConnectionEstablished()
+        {
+            ConnectionScreenController.OnSuccess();
```

Comment:
would move that to the end of the method so that the level and systems are already loaded / ready before the screen gets disabled.